sesman/chansrv/chansrv_fuse.c

```diff
@@ -265,7 +272,9 @@ static void xfuse_create_file(fuse_req_t req, fuse_ino_t parent,
 #endif
 
 static void xfuse_dump_fs(void);
+#if 0
```

Comment:
Please don't do it. It the code is not needed, please remove it. Don't let it rot. If the commented out code is needed (e.g. it illustrates something), please also add an explanation to the code.